test/test_gpu/skip_tests_h100_pytorch.yaml

```diff
@@ -29,9 +32,6 @@ jagged_layer_norm:
 jagged_mean:
 jagged_softmax:
 jagged_sum:
-layer_norm:
-low_mem_dropout:
-rms_norm:
```

Comment:
Liger kernels require triton package rather than pytorch-triton. I assume triton is not conflict with pytorch-triton because pytorch-triton doesn't cover `import triton`. I tested in local environment and it works well. but not sure if this is a safe way to do so.